Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/inbound/deferred/response/StandardInboundDocSubmissionDeferredResponse.java

```diff
@@ -122,14 +117,14 @@ private boolean isPolicyValid(RegistryResponseType request, AssertionType assert
         String senderHCID = assertion.getHomeCommunity().getHomeCommunityId();
 
         return policyChecker.checkXDRResponsePolicy(request, assertion, senderHCID, receiverHCID,
-                NhincConstants.POLICYENGINE_INBOUND_DIRECTION);
+            NhincConstants.POLICYENGINE_INBOUND_DIRECTION);
     }
 
     private String getLocalHCID() {
         String localHCID = null;
         try {
             localHCID = propertyAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE,
-                    NhincConstants.HOME_COMMUNITY_ID_PROPERTY);
+                NhincConstants.HOME_COMMUNITY_ID_PROPERTY);
         } catch (PropertyAccessException ex) {
             LOG.error("Exception while retrieving home community ID", ex);
         }
```

Comment:
Can we update the old code with the exception logging convention we are following now.
LOG.error("Exception while retrieving home community ID: " + ex.getLocalizedMessage(), ex);